### ISSUE TITLE
Generate a sample with 10-15 hospitals in the LA area

### ISSUE THREAD

**Issue by rileeki** [orchid-initiative/synthetic-database-project#89]
Harbor-UCLA
UCLA
LA General Medical Center
Keck Hospital 
Cedars Sinai
White Memorial
Glendale Adventist
Verdugo Hospital
Arcadia USC
Good Samaritan Hospital 
"maybe add a Kaiser too"

Check if you can add the two fields for facility ID to the hospital file for later post processing in format operations, or if that will mess with Synthea

**Comment by rileeki** [comment]
Check if you can add fields to the hospital file, or if that will mess with Synthea

**Comment by TravisHaussler** [comment]
I made an override file with these:
Harbor-UCLA
RONALD REAGAN UCLA
LA General Medical Center
Keck Hospital
Cedars Sinai
White Memorial
Glendale Adventist
Verdugo Hospital
Good Samaritan Hospital
Arcadia USC
Kaiser - Los Angeles

I am looking for the HCAI numbers now

**Comment by TravisHaussler** [comment]
For Good Samaritan Hospital is it the one in LA or the one in Bakersfield?  I wasn't sure if that's the hospital we are working with and wanted it in the list for some reason or if we just want the big LA one

**Comment by TravisHaussler** [comment]
Ok, so this works, yay!  I have output that has the correct facility IDs now.
One hiccup though - I ran it for population 10 and got 54 patients (which is a separate issue I referenced on the call last week) with 3449 encounters.

Of these encounters, we had 27 that met the requirement to be reported in our output (i.e. filtered for type inpatient).  And of those 27, 25 were in a hospital from our list.  One of the ones that was not from a hospital was in a "primary_care_facility" from another input file for Synthea.  

Diagnosis code was: 
`32485007	Hospital admission (procedure)`
Procedure codes were:
```
225338004	Risk assessment (procedure)
33367005	Angiography of coronary artery (procedure)
415070008	Percutaneous coronary intervention (procedure)
433236007	Transthoracic echocardiography (procedure)
```
